models/repo/repo.go

```diff
@@ -163,6 +163,7 @@ type Repository struct {
 	IsTemplate                      bool               `xorm:"INDEX NOT NULL DEFAULT false"`
 	TemplateID                      int64              `xorm:"INDEX"`
 	Size                            int64              `xorm:"NOT NULL DEFAULT 0"`
```

Comment:
Maybe we should have some comments here about what size means? Is it the total size or the size which excluded lfs size?

Reply:
```suggestion
	// the size of git repository directory
	Size                            int64              `xorm:"NOT NULL DEFAULT 0"`
```

how about this?

Reply:
```suggestion
        // the size of git repository directory itself, not include lfs/package/attachment size
	Size                            int64              `xorm:"NOT NULL DEFAULT 0"`
```